src/stores/__tests__/prefs-store.spec.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import PrefsStore from '../prefs-store'
+
+describe('PrefsStore', () => {
+  describe('hydrates', () => {
+    xit('default values', () => {})
```

Comment:
I disagree with this. xit/pending tests are exactly made for this - to show which edge cases/parts of the code are significant but still untested.

I'm going to be implementing tests for it anyway so I guess it's a non-issue 👍